charts/grafana/templates/servicemonitor.yaml

```
@@ -5,7 +5,7 @@ kind: ServiceMonitor
 metadata:
   name: {{ template "grafana.fullname" . }}
   {{- if .Values.serviceMonitor.namespace }}
-  namespace: {{ .Values.serviceMonitor.namespace }}
+  namespace: {{ tpl (.Values.serviceMonitor.namespace) . }}
```

Extra bracket is not needed.

```suggestion
  namespace: {{ tpl .Values.serviceMonitor.namespace . }}
```